### PR TITLE
Change button input for Comment section

### PR DESCRIPTION
**Pull request by MauroGuerrero20**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -70,7 +72,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(commentsQuery);
 
     for (Entity entity : results.asIterable()){
-      commentsList.add((String) entity.getProperty("comment"));
+      commentsList.add((String) entity.getProperty("comment_msg"));
```
Comment (VivekShankar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:75]: Good catch! I'm guessing this was responsible for the "null" comments :)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -63,4 +63,13 @@ function getComments() {
 window.onload = function() {
   randQuote();
   getComments();
-}
\ No newline at end of file
+}
+
+// Listens for Comments section button
+window.addEventListener("load", function() {
```
Comment (VivekShankar) [inline portfolio/src/main/webapp/script.js:69]: Is there any reason you didn't add the logic in this function directly to window's onload attribute (along with `randQuote` and `getComments`?

**Review by VivekShankar — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (MauroGuerrero20) [reply portfolio/src/main/webapp/script.js:69]: I forgot to optimize that implementation in this PR. The next PR should have ```randQuote``` and ```getComments``` in the same block.

**Pull request merged**